CHANGELOG.md

```diff
@@ -0,0 +1,79 @@
+v 0.3 API changes:
+
+C library:
```

Comment:
Other API changes have been made, e.g. you replaced an old function with `nu_masses()`.

Reply:
Thanks, I forgot to document this. Added now.